test/validation/api/classification/odp_classification_test_pmr.c

```diff
@@ -2406,6 +2406,250 @@ static void test_pmr_series(const int num_udp, int marking)
 	odp_pktio_close(pktio);
 }
 
+static void classification_test_pmr_term_sctp_sport(void)
```

Comment:
The sport and dport functions are almost exactly the same, differing only in a couple of lines. Please combine the content into single function that takes the port (src or dst) to be tested as a parameter. The original functions that are called by CUnit would reduce to simple wrappers that call the common function. This improves future maintainability of the code.

My other comments in this function apply also to the dport test function.

Reply:
Ack